### ISSUE TITLE
Fix a11y add-on

### ISSUE THREAD

**Issue by aVileBroker** [Headstorm/foundry-ui#20]
Perhaps because of styled-components, the a11y add-on for storybook doesn't work as it does with the angular version of hs-ui.

A cursory google search didn't find any explanation, so there could be another issue.

AC:
* The a11y add-on should be functioning and show passing/failing accessibility tests
